samples/mqtt_example/cloudiot_mqtt_example_nodejs.js

```diff
@@ -54,6 +52,15 @@ const createJwt = (projectId, privateKeyFile, algorithm) => {
 };
 // [END iot_mqtt_jwt]
 
+//iot-sandbox.clearblade.com
+const hostFormatted = hostRegion => {
+  if (hostRegion === 'us-central1') return 'us-central1-mqtt.clearblade.com';
+  else if (hostRegion === 'europe-west1')
+    return 'europe-west1-mqtt.clearblade.com';
+  else if (hostRegion === 'asia-east1') return 'asia-east1-mqtt.clearblade.com';
+  else return '';
```

Comment:
Should we throw an error here instead of returning an empty string? That way we fail early